### DISCUSSION TITLE
Adding padding_y for `TextBox` widget

### DISCUSSION THREAD

**Discussion by nullbyto** [qtile/qtile#3730] [Ideas] [answered]
When using `TextBox` sometimes some glyphs/icons come a little shifted vertically from the center. Therefore, `padding_y` key would fix that manually.
Margin options are useful as well.

**Comment by elParaguayo** [comment, marked as answer]
I started looking at this a while ago (https://github.com/qtile/qtile/pull/2346). Maybe we need to revisit it.

**Reply by nullbyto** [reply]
That would be awesome.
